os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -687,6 +687,29 @@ static inline void up_set_mpu_app_configuration(struct tcb_s *rtcb)
 #define up_set_mpu_app_configuration(x)
 #endif
 
+#if defined(CONFIG_MPU_STACKGUARD)
+static inline void up_set_mpu_region_configuration(struct tcb_s *rtcb, int region_no)
```

Comment:
1. Could you rename this function? I can't know what the difference against other mpu function.
2. Other mpu function has the region number as 1st argument. But this isn't. And it has different argument name for same purpose. Could you use the same function name policy?

Reply:
done